### PR TITLE
Fixed flaky PrintPreviewDialog tests and added a new test

### LINKED ISSUE #6501: Rewritten MAUI tests can't pass pipelines
* .NET Core Version: 
7.0.0-alpha.1.21576.4

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**
Some rewritten MAUI tests can't pass pipelines for some reasons. They're passing local build and staying successful for many reruns without any single fail. Unfortunately, in this case it's spoil debugging and for checking every test fix we should rerun pipelines, which many times less productive than rerun single problematic test locally. I suggest list of problematic tests below. The list could be updated in case of discovering new problematic tests.

- [ ] System.Windows.Forms.UITests.PrintPreviewDialogTests.PrintPreviewDialog_Hotkey_Ctrl_1Async (#8430)
- [ ] System.Windows.Forms.UITests.PrintPreviewDialogTests.PrintPreviewDialog_Hotkey_Ctrl_2Async (#8430)
- [ ] System.Windows.Forms.UITests.PrintPreviewDialogTests.PrintPreviewDialog_Hotkey_Ctrl_3Async (#8430)
- [ ] System.Windows.Forms.UITests.PrintPreviewDialogTests.PrintPreviewDialog_Hotkey_Ctrl_4Async (#8430)
- [ ] System.Windows.Forms.UITests.PrintPreviewDialogTests.PrintPreviewDialog_Hotkey_Ctrl_5Async (#8430)
- [x] System.Windows.Forms.UITests.ComboBoxTests.ComboBox_Select_Item_By_UpArrowKeyAsync (#7323)
- [x] System.Windows.Forms.Tests.PropertyGridView_PropertyGridViewAccessibleObjectTests.PropertyGridViewRowsAccessibleObject_Ctor_Default (#6405 )

### PR DESCRIPTION
**Pull request by dkazennov**
Fixes #6501

## Proposed changes

- added new stable unit test PrintPreviewDialog_Hotkey_Ctrl_Digit without InputSimulator.SendAsync;
- removed old flacky integration tests PrintPreviewDialog_Hotkey_Ctrl_1Async, PrintPreviewDialog_Hotkey_Ctrl_2Async, PrintPreviewDialog_Hotkey_Ctrl_3Async, PrintPreviewDialog_Hotkey_Ctrl_4Async and PrintPreviewDialog_Hotkey_Ctrl_5Async  in UIIntegrationTests
- added additional proposed test PrintPreviewDialog_Hotkey_ArrowKeys

## Customer Impact
- PrintPreviewDialogTests are replaced with the new stable test. No flaky test issue anymore.

## Regression? 

- No

## Risk

- Minimal

## Test methodology 

- Unit tests

## Test environment(s) 

- dotnet 8.0.100-alpha.1.22607.6

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/8430)

### CODE REVIEW

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        [WinFormsTheory]
+        [InlineData(Keys.D1, 1, 1)]
+        [InlineData(Keys.D2, 1, 2)]
+        [InlineData(Keys.D3, 1, 3)]
+        [InlineData(Keys.D4, 2, 2)]
+        [InlineData(Keys.D5, 2, 3)]
+        public void PrintPreviewDialog_Hotkey_Ctrl_Digit(Keys digitKey, int rows, int columns)
+        {
+            using TestPrintPreviewDialog testPrintPreviewDialog = new();
+            testPrintPreviewDialog.CreateControl(true);
```
[the remark this answers is not in the document]
Comment (dkazennov) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs:20]: Yes, indeed.

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        [InlineData(Keys.D2, 1, 2)]
+        [InlineData(Keys.D3, 1, 3)]
+        [InlineData(Keys.D4, 2, 2)]
+        [InlineData(Keys.D5, 2, 3)]
+        public void PrintPreviewDialog_Hotkey_Ctrl_Digit(Keys digitKey, int rows, int columns)
+        {
+            using TestPrintPreviewDialog testPrintPreviewDialog = new();
+            testPrintPreviewDialog.TestProcessDialogKey(Keys.Control | digitKey);
+
+            Assert.Equal(rows, testPrintPreviewDialog.PrintPreviewControl.Rows);
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs:22]: can you add tests for following aswell?
 case Keys.Left:
case Keys.Right:
 case Keys.Up:
case Keys.Down:

Comment (dkazennov) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs:22]: > can you add tests for following aswell? case Keys.Left: case Keys.Right: case Keys.Up: case Keys.Down:

Yes, there can be test to check that those keys return null.

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            Assert.Equal(columns, testPrintPreviewDialog.PrintPreviewControl.Columns);
+        }
+
+        [WinFormsTheory]
+        [InlineData(Keys.Left)]
+        [InlineData(Keys.Right)]
+        [InlineData(Keys.Up)]
+        [InlineData(Keys.Down)]
+        public void PrintPreviewDialog_Hotkey_ArrowKeys(Keys arrowKey)
```
Comment (dmitrii-drobotov) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs:31]: Nit: names can be changed to be more consistent with other unit tests and specify tested method and expected result, e.g. `PrintPreviewDialog_ProcessDialogKey_ArrowKeys_ReturnsFalse` and `PrintPreviewDialog_ProcessDialogKey_CtrlDigit_ChangesRowsAndColumnsCount`.

Comment (dkazennov) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewDialogTests.cs:31]: Yes, this looks move comprehensible. Let's do it.

**Review by dmitrii-drobotov — APPROVED** [review]

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**